packages/rocketchat-livechat/.app/client/views/messageAttachment.js

```diff
@@ -0,0 +1,32 @@
+import visitor from '../../imports/client/visitor';
+
+const fixLivechatUpload = function(url) {
+    let livechatUrl = url.replace('file-upload', 'livechat-file-upload');
```

Comment:
this adds a strong dependency on rocket.chat's upload URLs schema that is hard to track.. but we'll have to deal with it if we don't find a better way to handle it.

Reply:
I changed this behavior, please take a look at here:
https://github.com/RocketChat/Rocket.Chat/pull/10514/files#r200442868